pkg/cloud/provider.go

```diff
@@ -70,6 +70,18 @@ func (n *Node) IsSpot() bool {
 	}
 }
 
+// LoadBalancer is the interface by which the provider and cost model communicate LoadBalancer prices.
+// The provider will best-effort try to fill out this struct.
+type LoadBalancer struct {
+	IngressIPAddresses           []string `json:"IngressIPAddresses"`
```

Comment:
The interface should probably be a bit more general, with "cost" representing an hourly cost that takes into account the specifics of GCP loadbalancers. A function for computing cost probably needs to be provider specific, ie in gcpprovider.go. So in provider.go you may want a constructor-like function LoadBalancer) that takes in a service and returns a LoadBalancer with a "Cost" field calculated.